jib-cli/src/main/java/com/google/cloud/tools/jib/cli/Jar.java

```diff
@@ -67,11 +78,72 @@
 
   @CommandLine.Option(
       names = "--from",
-      paramLabel = "<from>",
+      paramLabel = "<base-image>",
       description = "The base image to use.")
   @SuppressWarnings("NullAway.Init") // initialized by picocli
   private String from;
 
+  @CommandLine.Option(
+      names = "--jvm-flags",
+      paramLabel = "<jvm-flags>",
+      split = ",",
+      description = "JVM arguments, example: --jvm-flags=-Dmy.property=value,-Xshare:off")
+  private List<String> jvmFlags = Collections.emptyList();
+
+  @CommandLine.Option(
+      names = "--expose",
+      paramLabel = "<exposed-ports>",
+      split = ",",
+      description = "Ports to expose on container, example: --expose=5000,7/udp.")
+  private List<String> exposedPorts = Collections.emptyList();
+
+  @CommandLine.Option(
+      names = "--volumes",
+      paramLabel = "<volumes>",
+      split = ",",
+      description =
+          "Directories on container to hold extra volumes,  example: --volumes=/var/log,/var/log2.")
+  private List<String> volumes = Collections.emptyList();
+
+  @CommandLine.Option(
+      names = "--environment-variables",
+      paramLabel = "<environment-variables>",
+      split = ",",
+      description =
+          "Environment variables to write into container, example: --environment-variables env1=env_value1,env2=env_value2.")
+  private Map<String, String> environment = new LinkedHashMap<>();
```

Comment:
Perhaps for consistency, `Collections.emptyMap()`. If `Collections.emptyList()` worked for others, `emptyMap()` should work. It's certain that no one is modifying the initially assigned collection, because `Collections.empty???()` are immutable. (They throw an exception if someone tries to modify it.)